utils/zerovec/derive/src/make_ule.rs

```diff
@@ -275,6 +275,7 @@ fn make_ule_struct_impl(
         #[repr(#repr_attr)]
         #[derive(Copy, Clone, PartialEq, Eq)]
         #[doc = #doc]
+        #[allow(missing_docs)]
```

Comment:
Observation: Hmm, there is a `#[doc]` line right above this one. Shouldn't it always be documented?

Reply:
The macro uses `#[doc]` to create doc at the struct level (for the struct that the macro generates), but the doc warnings come from all of the fields in the auto-generated struct. @Manishearth and I discussed and thought those fields are not useful to document.

Reply:
Nit: Please explain this in a comment in the code.